pages/documentation.html

```diff
@@ -30,50 +37,150 @@
 
 </div>
 
+<div class="framework">
+<div class="frameworklabel">Books and long tutorials</div>
+<div class="frameworkcontent">
 
+<p>The following books and long tutorials were written by experimented
+Coq users and teachers:</p>
 
-<div class="framework">
-<div class="frameworklabel">Other documentation</div>
+<ul>
 
-<div class="frameworkcontent">
-<p>Moreover, some users of Coq have published various documents that may be useful both to beginners and advanced users of Coq:</p> 
+<li>the <a href="http://www.labri.fr/perso/casteran/CoqArt/">Coq'Art</a>,
+the first book dedicated to the Coq proof assistant (Yves Bertot,
+Pierre Castéran, 2004, Chinese version in 2009), only the French
+version and the exercises can be downloaded for free;</li>
+
+<li><a href="http://www.cis.upenn.edu/~bcpierce/sf/">Software
+Foundations</a>, a series of Coq-based textbooks on logic, functional
+programming and foundations of programming languages (Benjamin
+Pierce <em>et al</em>, 2007, with regular updates), much acclaimed for
+being accessible to beginners, but rather oriented to computer
+scientists;</li>
+
+<li><a href="http://adam.chlipala.net/cpdt/">Certified Programming
+with Dependent Types</a>, a textbook about practical engineering with
+Coq (Adam Chlipala, 2008), teaches advanced practical tricks and a
+very specific style of proof;</li>
+
+<li>the <a href="https://math-comp.github.io/mcb/">Mathematical
+Components book</a> (A. Mahboubi and E. Tassi, 2018), a book that is
+more oriented to mathematical inclined users, to dive into Coq with
+the SSReflect proof language, and the Mathematical Components
+library.</li>
+
```

Comment:
Here is a suggested addition for FRAP:
```suggestion
<li><a href="http://adam.chlipala.net/frap/">Formal Reasoning About Programs</a> (Adam Chlipala, 2017), 
a book that simultaneously provides a general introduction to formal logical reasoning about the correctness of 
program and to using Coq for this purpose.</li>
```
Here is a suggested addition for PnP:
```suggestion
<li><a href="https://ilyasergey.net/pnp/">Programs and Proofs</a>, (Ilya Sergey, 2017), a book that gives a
brief and practically-oriented introduction to the basic concepts of interactive theorem proving using Coq; 
emphasizes the computational nature of inductive reasoning about decidable propositions via a small set of 
primitives from the SSreflect proof language.</li>
```

Reply:
I think "program" should be "programs" in the FRAP text.

Reply:
Thanks for the catch, fixed now.